media/css/firefox/privacy/promise.scss

```
@@ -7,70 +7,6 @@ $image-path: '/media/protocol/img';
 
 @import '~@mozilla-protocol/core/protocol/css/includes/lib';
 
-// * -------------------------------------------------------------------------- */
-// Main Hero
-
-.privacy-promise-hero {
-    p {
-        @include text-body-lg;
-    }
-
-    .mzp-c-split-media-asset {
-        padding-top: $spacing-xl;
-    }
-
-    @media #{$mq-md} {
-        .mzp-c-split-bg {
-            padding-top: 0;
-            padding-bottom: 0;
-        }
-
-        .mzp-c-split-body {
-            padding-top: $spacing-2xl;
-            padding-bottom: $spacing-2xl;
-        }
-    }
-}
-
-// * -------------------------------------------------------------------------- */
-// Data Promise Call Out
-
-.call-out-data-promise {
-    background: $color-white;
-
-    .mzp-c-callout-title {
-        @include text-title-sm;
-    }
-}
-
-// * -------------------------------------------------------------------------- */
-// Privacy Promise Splits
-
-.privacy-promise-feature {
-    h2 {
-        color: $color-violet-70;
-    }
-
-    .privacy-promise-sub-title {
-        @include text-title-xs;
-    }
-}
-
-.mzp-c-split-media {
-    max-width: 100%;
-}
-
-// * -------------------------------------------------------------------------- */
-// Picto Cards
-
-.mzp-l-columns .mzp-c-picto {
-    margin-bottom: $layout-md;
-    @media #{$mq-md} {
-        margin-bottom: 0;
-        padding: $spacing-2xl $spacing-2xl 0;
-    }
-}
-
-.privacy-promise-learn-more {
-    background-color: $color-marketing-gray-10;
+h1{
+    @include text-title-xl;
```

This all looks very good, just one thing I noticed is now the specificity for h1 is not enough, so maybe adding the class too to override the generic picto sizing?

```suggestion
h1.mzp-c-picto-heading {
    @include text-title-xl;
```